dxr/plugins/rust/tests/__init__.py

```diff
@@ -0,0 +1,15 @@
+from dxr.testing import SingleFileTestCase
+
+class RustSingleFileTestCase(SingleFileTestCase):
+    """Test case suited to testing the Rust plugin."""
+    source_filename = 'mod.rs'
+
+    @classmethod
+    def config_input(cls, config_dir_path):
+        config = super(RustSingleFileTestCase, cls).config_input(config_dir_path)
+
+        config['DXR']['enabled_plugins'] = 'pygmentize rust'
+        config['DXR']['disabled_plugins'] = 'clang'
+        config['code']['build_command'] = '$RUSTC mod.rs'
```

Comment:
On the Vagrant box, $RUSTC is empty, so we'll need to rig a way to fall back to "rustc" like the indexing plugin does.